src/components/common/atoms/Avatar/index.tsx

```diff
@@ -0,0 +1,13 @@
+import { AvatarImg } from './styled';
+
+type Props = {
+  handleClick: () => void;
+  pic: string;
+  className?: string;
+};
+
+const Avatar = ({ handleClick, pic, className }: Props) => {
+  return <AvatarImg src={pic} onClick={handleClick} className={className} />;
```

Comment:
Alternatively, it could also be:
`const Avatar = ({ handleClick, pic, className }: Props) => <AvatarImg src={pic} onClick={handleClick} className={className} />;`